$NetBSD: patch-ae,v 1.1 2011/10/03 23:45:27 schnoebe Exp $

Conditionally use AI_ADDRCONFIG, it doesn't exist on all platforms.

--- cfg.c.orig	2010-08-26 20:21:25.000000000 +0000
+++ cfg.c
@@ -122,7 +122,10 @@ void cfg_addaddress(struct cvsd_cfg *cfg
 
   /* create the name for the socket */
   memset(&hints,0,sizeof(struct addrinfo));
-  hints.ai_flags=AI_PASSIVE|AI_ADDRCONFIG;
+  hints.ai_flags=AI_PASSIVE;
+#if defined(AI_ADDRCONFIG)	/* a linux-ism */
+  hints.ai_flags|=AI_ADDRCONFIG;
+#endif
   hints.ai_family=PF_UNSPEC;
   hints.ai_socktype=SOCK_STREAM;
   switch (i=getaddrinfo(node,service,&hints,&(addr->addrs)))
